### DISCUSSION TITLE
how to make background transparency in alacritty

### DISCUSSION THREAD

**Discussion by WingDust** [NvChad/NvChad#1204] [Q&A] [answered]
I already write 
```lua
vim.cmd([[
func! s:transparent_background()
    highlight Normal guibg=None ctermbg=None
    highlight NonText guibg=None ctermbg=None
endf
autocmd ColorScheme * call s:transparent_background()
]]
)
```
in chadrc.lua 
with follow https://github.com/alacritty/alacritty/issues/1082,
but no effect

**Comment by siduck** [comment]
pls use the chadrc option :<

**Reply by searleser97** [reply]
could you clarify ? or be more explicit on your answer ? I do not know anything about the chadrc option, Where can I find info about it ?

**Reply by siduck** [reply, marked as answer]
@searleser97 there's already a transparency option, please read core/default_config.lua

**Comment by QieQiufeng** [comment]
![image](https://github.com/NvChad/NvChad/assets/82828246/3aff38fc-acad-4fe0-bc62-8b2c6f85c1b5)
哈哈，超级简单，感谢大佬！
效果如下：
![image](https://github.com/NvChad/NvChad/assets/82828246/724ebe17-6073-46fe-b75c-24d01a35ed21)

**Comment by zougari47** [comment]
For me it work in `base64` table:
```lua
M.base46 = {
  theme = "tomorrow_night",
  transparency = true,
  -- hl_override = {
  -- 	Comment = { italic = true },
  -- 	["@comment"] = { italic = true },
  -- },
}
```
NvChad 2.5 , NVim 0.12.0

**Reply by siduck** [reply]
@zougari47 yes, since last year theme related options have been moved to base46 table. :h nvui is your friend :D
